src/setup-server/setup-server.ts

```diff
@@ -11,7 +12,8 @@ export const setupServer = async () => {
       return {
         auth
       };
-    }
+    },
+    formatError: formatError
```

Comment:
```suggestion
    formatError,
```
dica: quando você está declarando um objeto chave valor, se o valor está numa variável que tem o mesmo nome da chave (como o caso do `formatError` aqui, dá pra omitir o valor e deixar só `chave,`